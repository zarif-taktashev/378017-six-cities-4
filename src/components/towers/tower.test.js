import React from "react";
import renderer from "react-test-renderer";
import Tower from "./towers.jsx";

const offers = [`Amsterdam`, `Paris`];

describe(`Towers-test`, () => {
  it(`Towers unit test`, () => {
    const tree = renderer
      .create(<Tower
        towers={offers}
        onSelectCity={() => {}}
        activeCity={`Paris`}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
