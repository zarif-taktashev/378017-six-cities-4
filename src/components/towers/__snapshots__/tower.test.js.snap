// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Towers-test Towers unit test 1`] = `
<ul
  className="locations__list tabs__list"
  onClick={[Function]}
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Paris
      </span>
    </a>
  </li>
</ul>
`;
